src/System.Net.Http.WinHttpHandler/src/project.json

```diff
@@ -2,7 +2,9 @@
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.1-rc3-23908",
     "Microsoft.Win32.Primitives": "4.0.0",
+    "System.Collections": "4.0.0",
     "System.Diagnostics.DiagnosticSource": "4.0.0-rc3-23908",
+    "System.Diagnostics.Debug": "4.0.10",
```

Comment:
When I switched to netstandard I get different dependencies since with fewer transitive impl dependencies leaking into compile graph.